### PR TITLE
React query

### PR DESCRIPTION
**Pull request by jessecoleman**
- new data fetching layer using react-query
- refactor all endpoints to be `GET` instead of `POST`
- redux is GONE
- chart settings now expands as popup menu with more settings
- default to showing all divisions on page load
- elasticsearch aggregate query now returns hierarchical structure directly
- general indicators for loading states

### CODE REVIEW

**Comment by h1-the-swan** [comment]
Oh you got rid of Redux? I thought you were doubling down with RTK Query

File: frontend/package.json [hunk 1]
```diff
@@ -20,34 +20,34 @@
...
-        "redux-logger": "^3.0.6",
         "typescript": "^4.2.4",
         "use-query-params": "^1.2.3"
     },
     "scripts": {
         "start": "react-scripts start",
         "build": "react-scripts build",
-        "generateAndStart": "node ./scripts/generateAPI.mjs && yarn start",
+        "generate": "orval --config ../orval.config.js",
+        "generateAndStart": "orval --config ../orval.config.js && yarn start",
```
Comment (h1-the-swan) [inline frontend/package.json:50]: This is not going to work with our current workflow. The reason I used the `generateAPI` script is that the backend and frontend are in two separate containers. They communicate with each other only through the API. The `generateAPI` script used an API call to retrieve the API schema from the backend. This method using `orval` would require direct access to the backend.

**Review by h1-the-swan — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 1 above)
Comment (jessecoleman) [reply frontend/package.json:50]: Without creating a lot of extra work, is it possible to have the python app spit out the API.json as part of the build step of the frontend? I guess the frontend docker would need the python runtime

**Review by h1-the-swan — APPROVED** [review]
It's looking good. Everything seems to be working properly using docker-compose for development.

**Pull request merged**